File: play-2.4.x/K-Modules/kmodules/K-Base/app/models/SampleModel.java
package models;

import com.avaje.ebean.Model;
import jodd.datetime.JDateTime;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;

/**
 * Created by kk on 15/7/11.
 */

@Entity
public class SampleModel extends Model {

    @Id
    public long id;

    @Column(columnDefinition = "DATETIME COMMENT '任务计划执行时间'")
    public JDateTime plan_run_time;

    public static Finder<Long, SampleModel> find = new Finder<>(SampleModel.class);
}

File: play-kotlin-2.5.x/PlayKotlin/kmodules/K-Base/app/models/SampleModel.kt
package models

import com.avaje.ebean.Model
import jodd.datetime.JDateTime

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id

/**
 * Created by kk on 15/7/11.
 */

@Entity
class SampleModel : Model() {

    @Id
    var id: Long = 0

    @Column(columnDefinition = "DATETIME COMMENT '任务计划执行时间'")
    var plan_run_time: JDateTime? = null

    companion object {

        var find = Model.Finder<Long, SampleModel>(SampleModel::class.java)
    }
}
